README.md

```diff
@@ -76,7 +76,7 @@ For example, using VirtualBox:
 vagrant --instances=2 --repo=/home/jdoe/keylime --cpus=4 --memory=4096  up --provider virtualbox --provision
 ```
 
-| NOTE: Customized args (`--instances`, `--repos` etc), come before the mainvagrant args (such as `up`, `--provider`) |
+| NOTE: Customized args (`--instances`, `--repos` etc), come before the mainvagrant args (such as `up`, `status`, `--provider`). Example: To `ssh` into the second machine instance, keylime2, use the vagrant command as such : `vagrant --instances=2 ssh keylime2` |
```

Comment:
That's interesting, do you really need the `--instances=2` flag? Can you not just do `vagrant ssh keylime2`?

Reply:
Directly running `vagrant ssh keylime2` doesn't work unless the `--instances=2` option is specified. That's probably because of the defaults set in our Vagrantfile [here](https://github.com/keylime/ansible-keylime-tpm-emulator/blob/0f30b1c5b3a7724553639166c131bb36956cf00e/Vagrantfile#L14).

Reply:
That's interesting, I guess its from this loop:

https://github.com/keylime/ansible-keylime-tpm-emulator/blob/0f30b1c5b3a7724553639166c131bb36956cf00e/Vagrantfile#L38-L39

I will merge this and we can try to work on fixing this. It ruby based, so we might be able to work around this somehow.

Reply:
Yup, the `--instances` argument then reaches that loop, and it uses the defaults if it isn't specified.
Yeah sure, I'll keep trying ways to configure it better in the meantime.